.gitignore

```diff
@@ -1,3 +1,4 @@
 bundle
 bundle.zip
 exampleplugin
+**/.DS_Store
```

Comment:
I think this should go in your global .gitignore

Reply:
it should - but that doesn't help when others inevitably push one, right?